docs/_ext/copy_sphinx_gallery_notebooks.py

```diff
@@ -0,0 +1,76 @@
+import os
```

Comment:
General remark: Although they are not part of the `skactiveml` package, it would be great to provide (very) brief documentation for these functions. This way, we ensure easier understanding in the future.